cmd/koordlet/main.go

```diff
@@ -24,6 +24,7 @@ import (
 
 	"github.com/prometheus/client_golang/prometheus/promhttp"
 	"k8s.io/apimachinery/pkg/util/wait"
+	_ "k8s.io/component-base/logs"
```

Comment:
Because the new code indirectly depends on the logs package, it will conflict with the original klog.InitFlags. So it needs to be handled like this.